designs/dragonphy_top/qtm/termination.qtm.tcl

```diff
@@ -12,15 +12,21 @@ set_qtm_technology -library $::env(qtm_tech_lib)
 create_qtm_drive_type -lib_cell $ADK_DRIVING_CELL qtm_drive
 create_qtm_load_type -lib_cell $ADK_DRIVING_CELL qtm_load
 
-# define inputs
+# define input(s)
 set input_list { \
- VinP \
- VinN \
- Vcm \
+    VinP \
+    VinN \
 }
 create_qtm_port $input_list -type input
 set_qtm_port_load -type qtm_load -factor 2 $input_list
 
+# define output(s)
+set output_list { \
+    Vcm \
```

Comment:
The synthesis tool didn't like having an input driving an output (refused to wire it up).  It also didn't like having two ``inouts`` connected to a single output, so I settled on making ``Vcm`` an output and connecting it to ``DOUTP``/``DOUTN``.  It could be that making ``DOUTP``/``DOUTN`` ``inouts`` would work too, but I haven't tried it yet.